src/locales/fr.json

```diff
@@ -925,6 +925,7 @@
   "SelectionBar": {
     "selected_count": "élément sélectionné |||| éléments sélectionnés",
     "close": "Fermer",
-    "categorize": "Categorisation"
+    "categorize": "Categorisation",
+    "selectAll": "Tout sélectionner"
```

Comment:
Do we really select all the transactions, or do we select only the displayed transactions?

Reply:
On ne sélectionne que celles affichées